Packaging.props

```diff
@@ -19,6 +19,7 @@
     <!-- temporary change until buildtools can be updated https://github.com/dotnet/buildtools/issues/831 -->
     <LineupPackageVersion Condition="'$(LineupPackageVersion)' == ''">1.0.3</LineupPackageVersion>
     <PlatformPackageVersion Condition="'$(PlatformPackageVersion)' == ''">1.0.2</PlatformPackageVersion>
+    <SkipValidatePackageTargetFramework>true</SkipValidatePackageTargetFramework>
```

Comment:
TL;DR - this validation was for bringing up generations/netstandard before it existed in NuGet.

We did our own graph walk of assembly dependencies to determine what generation/netstandard version was appropriate for an assembly.  We seeded this with all the predetermined assembly versions defined by previously shipped frameworks and the generations they mapped to.  This let us ensure we were putting assemblies in the appropriate generation.

Now that NuGet supports generations/netstandard and we have a stable set of packages with the netstandard API in it, we can use those instead of doing this validation.  I opened https://github.com/dotnet/corefx/issues/11009 to track this cleanup.